pyproject.toml

```diff
@@ -338,18 +319,37 @@ module = [
     "tests.test_util.test_util_docstrings",
     "tests.test_util.test_util_docutils",
     "tests.test_util.test_util_docutils_sphinx_directive",
+    "tests.test_util.test_util_fileutil",
+    "tests.test_util.test_util_i18n",
     "tests.test_util.test_util_images",
     "tests.test_util.test_util_importer",
+    "tests.test_util.test_util_inspect",
     "tests.test_util.test_util_inventory",
     "tests.test_util.test_util_lines",
+    "tests.test_util.test_util_logging",
     "tests.test_util.test_util_matching",
+    "tests.test_util.test_util_nodes",
+    "tests.test_util.test_util_rst",
+    "tests.test_util.test_util_template",
+    "tests.test_util.test_util_typing",
     "tests.test_util.test_util_uri",
     # tests/test_writers
     "tests.test_writers.test_api_translator",
     "tests.test_writers.test_docutilsconf",
     "tests.test_writers.test_writer_latex",
 ]
+check_untyped_defs = false
+disable_error_code = [
+    "annotation-unchecked",
+]
+disallow_incomplete_defs = false
```

Comment:
This (and the surrounding `= false` changes) reduces the type safety on a large number of files which are already complaint. How about making a new `[[tool.mypy.overrides]]` for the files which need these affordances?

Reply:
This was an effort to get to a passing state with a minimal-ish set of changes. I would prefer to put all of this in a `mypy.toml` file, but I don't think Mypy supports that -- and otherwise we have an even longer `pyproject.toml`.

Reply:
If this solution results in less mypy coverage then we shouldn't pursue it

Reply:
I've now split the override groups into the previously-ignored and the previously-excluded -- does this address your concern?

Reply:
It does for me! Hopefully we can reduce this list over time and address your concern of having such a long `pyproject.toml`.

Reply:
@danieleades I ran `mypy --html-report tests/` on master and this branch:

|  | Coverage | LOC |
|--------|--------|--------|
| Base | 15.00% imprecise | 101553 LOC |
| New | 23.11% imprecise | 133047 LOC | 

So 8.11pp more "imprecise", but 31% more lines checked / reported. Is this acceptable?

A